cmd/utils/okteto.go

```diff
@@ -53,6 +55,21 @@ func HasAccessToNamespace(ctx context.Context, namespace string, oktetoClient ty
 	return false, nil
 }
 
+func HasAccessToNamespace(ctx context.Context, namespace string, k8sClient kubernetes.Interface) (bool, error) {
+	nList, err := k8sClient.CoreV1().Namespaces().List(ctx, metav1.ListOptions{})
```

Comment:
listing namespaces requires admin access, the user might only have access to a single namespace